.github/workflows/ci.yml

```diff
@@ -61,79 +66,94 @@ jobs:
-            path: ${{ runner.temp }}\built_artifact
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup MSYS2
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: ${{ matrix.sys }}
+      - name: Shorten SHA signature
+        shell: msys2 {0}
+        run: echo "SHA7=`echo ${GITHUB_SHA} | cut -c1-7`" >> $GITHUB_ENV
+      - name: Install packages
+        shell: msys2 {0}
+        run: pacman -S --noconfirm mm-common autoconf automake gcc make cmake libtool
+      - name: Install dependencies
+        shell: msys2 {0}
+        working-directory: ${{ github.workspace }}
+        run: bash setup.sh
+      - name: Build Ruby ${{ matrix.ruby_ver }}
+        shell: msys2 {0}
+        working-directory: ${{ github.workspace }}
+        run: cd windows; make RUBY_VER=${{ matrix.ruby_ver }}
+      - name: Build ModShot
+        shell: msys2 {0}
+        working-directory: ${{ github.workspace }}
+        run: |
+          cd windows; source vars.sh; cd ..
+          meson build --prefix=$PWD/out -Dmri_version=${{ matrix.ruby_ver }} -Dsteam=${{ matrix.steam }}
+          cd build; ninja install
+      - name: Prepare artifact
+        run: |
+          mkdir ${{ runner.temp }}\artifact
+          move ${{ github.workspace }}\out\* ${{ runner.temp }}\artifact\
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ModShot_Windows_${{ env.SHA7 }}_Ruby-${{ matrix.ruby_ver }}-${{ matrix.sys }}_Steam-${{ matrix.steam }}
+          path: ${{ runner.temp }}\artifact
 
   release-dev:
-    name: Release ModShot Dev
+    name: Release ModShot (dev)
     needs: build-windows
     runs-on: ubuntu-latest
     if: ${{ !(startsWith(github.ref, 'refs/tags/v')) }}
     steps:
+      - name: Shorten SHA commit signature
+        run: echo "SHA7=`echo ${GITHUB_SHA} | cut -c1-7`" >> $GITHUB_ENV
+      - name: Create directory for artifacts
+        run: mkdir ${{ runner.temp }}/artifacts
       - name: Download artifacts
-        uses: dawidd6/action-download-artifact@v2
+        uses: actions/download-artifact@v3
         with:
-          workflow: ci.yml
-          skip_unpack: true
-          path: ${{ github.workspace }}
-          run_id: ${{ github.run_id }}
-          workflow_conclusion: in_progress
-      - name: Shorten SHA
-        id: sha
-        run: echo "::set-output name=sha_short::${GITHUB_SHA::7}"
-      - uses: "marvinpinto/action-automatic-releases@latest"
+          path: ${{ runner.temp }}/artifacts
+      - name: Make archives
+        working-directory: ${{ runner.temp }}
+        run: |
+          mkdir release
+          cd artifacts
+          for i in */; do (cd "$i"; zip -r "../../release/${i%/}.zip" .); done
+      - name: Release ModShot
+        uses: marvinpinto/action-automatic-releases@latest
         with:
-          repo_token: "${{ secrets.GITHUB_TOKEN }}"
-          automatic_release_tag: "latest"
```

Comment:
Any reason why the auto release tag isn't set for dev?

Reply:
Now it's set to `automatic_release_tag: latest-dev`, i'm right? or I'm missing something

Reply:
Yes

Reply:
Can you please rename it back to latest